sw/host/opentitanlib/src/app/config/hyperdebug_teacup.json

```diff
@@ -404,6 +404,16 @@
       "name": "VBUS_SENSE",
       "mode": "Input",
       "alias_of": "CN10_18"
+    },
+    {
+        // Connected to active-low shutdown pin on the LED driver. This is
+        // pulled low by the rev 1.5 Teacup board, so it must be driven high to
+        // activate the driver. This configuration makes a high output the
+        // default.
```

Comment:
I'll acknowledge ahead of time that comments are not valid JSON, but this file already includes comments documenting other aspects of the configuration, so I think we're not worse off by including this.